api/oss/src/core/mounts/service.py

```
@@ -443,14 +831,194 @@ async def list_files(
+            for obj in level_files:
+                rel = (
+                    obj.key[len(mount_base) :]
+                    if obj.key.startswith(mount_base)
+                    else obj.key
+                )
+                if not rel or not _keep(rel, False):
+                    continue
+                seen.add(rel)
+                shallow.append(MountFile(path=rel, size=obj.size, mtime=obj.mtime))
+            subdir_rels: List[str] = []
+            for sub in level_subdirs:
```

Flake8 (7.3.0)

[error] import 'sub' from line 5 shadowed by loop variable

(F402)